lib/net.js

```diff
@@ -788,7 +789,7 @@ function connect(self, address, port, addressType, localAddress, localPort) {
     err = bind(localAddress, localPort);
 
     if (err) {
-      var ex = exceptionWithHostPort(err, 'bind', localAddress, localPort);
+      ex = exceptionWithHostPort(err, 'bind', localAddress, localPort);
```

Comment:
Declaring the var upfront is not really necessary (it gets hoisted anyway) and it makes the diff more noisy than it really needs to be.